docs/guides/upgrading.rst

```diff
@@ -454,6 +454,11 @@ Also several workarounds for very old views are no longer performed. Make these
  - Set ``$vars['name']`` instead of ``$vars['internalname']``.
  - Set ``$vars['id']`` instead of ``$vars['internalid']``.
 
+Removed libraries
+-----------------
+
+ - ``elgg.markdown``: replace with ``use \Michelf\MarkdownExtra``
```

Comment:
Agree with Steve: let's not encourage folks to have indirect dependencies. Removing one of our requirements should be considered a BC change.